models/plan.py

```diff
@@ -111,3 +108,26 @@ def close_camps(self, *camps: Camp) -> None:
             raise self.MissingCampsError()
         else:
             self.save()
+
+    def close(self, __is_closed=None):
+        """
+        Set __is_closed flag to be True if plan is closed.
+        """
+        self.__is_closed = True
+        self.__close_date = datetime.today().date()
+        self.save()
+
+    @property
+    def close_date(self) -> date:
+        """
+        Get the read only close date of the plan.
+        """
+        if self.__is_closed:
+            return self.__close_date
+
+    @property
+    def close_plan_status(self):
```

Comment:
1. I suggest naming this `is_closed`
2. Can add the return type hint `-> bool`

Reply:
Ok, changed it now.